README.md

```diff
@@ -86,63 +70,51 @@ Prior to running this playbook some infrastructure needs to be in place
       /astores                      <- location for astores
   ```
 
-### Installation
+## Getting Started
+
+### Clone this project
 
-#### Ansible
+Run these commands in a Terminal session:
 
 ```bash
-# clone repo
-git clone https://github.com/sassoftware/viya4-deployment.git
+# clone this repo
+git clone https://github.com/sassoftware/viya4-deployment
 
 # move to directory
 cd viya4-deployment
-
-# install python packages
-pip3 install --user -r requirements.txt
-
-# install ansible collections
-ansible-galaxy collection install -r requirements.yaml -f
 ```
 
-#### Docker
+### Authenticating Ansible to access Cloud
```

Comment:
Authenticating Ansible to access Cloud Provider. 

** Note to developers. Our customer facing Viya Deployment Guide us the term 'cloud provider' vs simply cloud. To be consistent, we should as well.

Reply:
updated